## Задание 1

Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.

### Решение

$\overline{X} = M(X) = \frac{a + b}{2} = \frac{200 + 800}{2} = 500$ \
$D(X) = \frac{(b - a)^2}{12} = 30000$

## Задание 2

О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.

### Решение

$
\begin{cases}
\frac{(b - a)^2}{12} = 0.2 \\
a = 0.5
\end{cases}
\implies
b^2 - b + 0.25 - 2.4 = 0 \implies b^2 - b - 2.15 = 0$

$b1 = \frac{1 + \sqrt{9.6}}{2}$ \
$b2 = \frac{1 - \sqrt{9.6}}{2}$


In [12]:
b1 = (1 + 9.6**0.5)/2
b2 = (1 - 9.6**0.5)/2

(b1, b2)

(2.049193338482967, -1.0491933384829668)

Отрицательный корень отбрасываем. Ответ: 2.049

## Задание 3

Непрерывная случайная величина X распределена нормально и задана плотностью распределения \
f(x) = (1 / (4 * sqrt(2*pi))) * (exp(-(x+2)^2) / 32). \
Найдите: \
а). M(X) \
б). D(X) \
в). std(X) (среднее квадратичное отклонение)

### Решение

Общий вид распределения Гаусса:

$$
f(x)=\frac{1}{\sigma\cdot\sqrt{2\pi}}e^{-\frac{(x - a)^2}{2\sigma^2}}
$$

Отсюда 
$\sigma = 4$, $a = -2$ \
$b = a + 6\sigma = 22$ 

$M(X) = \frac{a + b}{2} = \frac{-2 + 22}{2} = 10 $ \
$D(X) = \sigma^2 = 16$ \
$std(X) = \sigma = 4$

## Задание 4

Рост взрослого населения города X имеет нормальное распределение. \
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см. \
Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост: \
а). больше 182 см \
б). больше 190 см \
в). от 166 см до 190 см \
г). от 166 см до 182 см \
д). от 158 см до 190 см \
е). не выше 150 см или не ниже 190 см \
ё). не выше 150 см или не ниже 198 см \
ж). ниже 166 см.

### Решение

In [35]:
a = (182 - 174) / 8 
b = (190 - 174) / 8
c = (166 - 174) / 8
d = (158 - 174) / 8
e = (150 - 174) / 8
f = (198 - 174) / 8
a, b, c, d, e, f

(1.0, 2.0, -1.0, -2.0, -3.0, 3.0)

In [36]:
pa = 0.3413
pb = 0.4773
pc = 0.1587
pd = 0.0228
pe = 0.0014
pf = 0.9986

In [31]:
# а
1 - pa


0.6587000000000001

In [32]:
# б
1 - pb

0.5226999999999999

In [37]:
# в
pb - pc

0.3186

In [38]:
# г
pa - pc

0.18259999999999998

In [40]:
# д
pb - pd

0.4545

In [45]:
# е
pe + (1 - pb)

0.5240999999999999

In [46]:
# ё
pe + (1 - pf)

0.002799999999999957

In [47]:
# ж
pc

0.1587

## Задание 5

На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от
математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

### Решение

$\sigma = \sqrt{25} = 5$

In [48]:
(190 - 178)/5 

2.4